swagger_spec_compatibility/rules/changed_additional_properties_to_false.py

```diff
@@ -34,5 +34,5 @@ def validate(cls, left_spec, right_spec):
         for additional_properties_diff in AdditionalPropertiesDifferWalker(left_spec, right_spec).walk():
             if additional_properties_diff.diff_type != DiffType.VALUE:
                 continue
-            if additional_properties_diff.additionalProperties.new == False:
+            if getattr(getattr(additional_properties_diff, 'additionalProperties', {}), 'new', {}) is False:
```

Comment:
Why do we need this? This defeats the checks provided by the type checker.
I would leverage the typing informations in this case rather than using `getattr` magic (which fully invalidates all the knowledge that `mypy` might have on the objects)

Reply:
Sure, I can do that